app/src/main/java/com/fbu/fbuteam/activities/HomeActivity.java

```diff
@@ -1,18 +1,102 @@
 package com.fbu.fbuteam.activities;
 
 import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
 
 import com.fbu.fbuteam.R;
+import com.fbu.fbuteam.fragments.NewsFragment;
+import com.fbu.fbuteam.fragments.ProfileFragment;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
 
 public class HomeActivity extends AppCompatActivity {
+    private Toolbar toolbar;
+    private BottomNavigationView bnvHome;
+    static MenuItem miActionProgressItem;
```

Comment:
change to `private static` and change variable name to `menuActionProgressItem` would be better